app/server/server.sbt

```diff
@@ -6,6 +6,8 @@ Compile / fork := true
 
 libraryDependencies ++= Deps.server(scalaVersion.value)
 
+version := "0.4.0"
```

Comment:
Can you check that this doesn't interfere with `sbt-cli-release`, i believe there is a requirement that the version number isn't set for our binaries to automatically publish to sonatype? https://github.com/olafurpg/sbt-ci-release

Reply:
Yep, had to remove this. I guess when doing the windows binary we have to temporarily set this. It might automatically do it correct when we have it at the v0.4.0 tag. I guess we'll see.

Reply:
Looks like it will set it properly when we are on the tag

https://github.com/dwijnand/sbt-dynver#detail